scripts/snippets/inf-hosp.r

```diff
@@ -0,0 +1,19 @@
+## first, choose random delays
+inf_hosp <- infection_times |>
+  mutate(
+    incubation_period = rgamma(n(), shape = 5, rate = 1),
+    onset_to_hosp = rlnorm(n(), meanlog = 1, sdlog = 1),
+    infection_date = as.Date(start_date + infection_time),
+    onset_date = as.Date(start_date + infection_time + incubation_period),
+    hosp_date =
+      as.Date(start_date + infection_time + incubation_period + onset_to_hosp)
+  )
+## next, set 70% of the hospitalization dates to NA because only 30∞ of cases
```

Comment:
Do you reeally need this complexity for the lesson at hand?

Reply:
Perhaps not - although it helps illustrate the point that doing so leads to a less sharp time distribution as in the figure that follows. That's a point I think we should try to get across. Do you have a suggestion for a better way of doing so?

Reply:
see https://github.com/nfidd/nfidd/pull/17#discussion_r1624199126